internal/user/repository.go

```diff
@@ -17,8 +17,8 @@ func NewRepository(db *sql.DB) *Repository {
 	return &Repository{db}
 }
 
-func (r *Repository) SaveTokenSignOn(email, token, userType string) error {
-	if _, err := r.db.Exec(`INSERT INTO user_sign_on_token (token, email, user_type) VALUES ($1, $2, $3)`, token, email, userType); err != nil {
+func (r *Repository) SaveTokenSignOn(email, token, userType string, createdAt time.Time) error {
```

Comment:
everything looks good, can you please just use `NOW()` in the query instead of passing downt he createdAt time? it's going to be less code and less prone to errors

Reply:
Gotcha